deploy/template.yaml

```diff
@@ -201,6 +201,8 @@ Mappings:
       MISSIONLABWEBSOCKETADDR: "wss://s9hlnv7y91.execute-api.eu-west-2.amazonaws.com"
       DEVICEINTELLIGENCETOGGLE: "0"
       BRANDREFRESHENABLED: "0"
+      FINGERPRINTCOOKIEDOMAIN: "account.gov.uk"
```

Comment:
Is it okay if it's the same on pre-prod envs or should it be more specific?

Reply:
Ah nevermind, just realised that was literally the whole point of this PR

Reply:
Haha yes it is!